gatsby-config.js

```diff
@@ -162,6 +143,53 @@ module.exports = {
             output: "/rss-contributors.xml",
             title: "Layer5"
           },
+          {
+            serialize: ({ query: { site, allPosts }}) => {
+              return allPosts.nodes.map( node => {
+                return Object.assign({}, node.frontmatter, {
+                  title: node.frontmatter.title,
+                  author: node.frontmatter.author,
+                  description: node.frontmatter.description,
+                  date: node.frontmatter.date,
+                  url: site.siteMetadata.siteUrl + node.fields.slug,
+                  guid: site.siteMetadata.siteUrl + node.fields.slug,
+                  enclosure: node.frontmatter.thumbnail && {
+                    url: site.siteMetadata.siteUrl + node.frontmatter.thumbnail.publicURL,
+                  },
+                  custom_elements: [{ "content:encoded": node.html }],
+                });
+              });
+            },
+            query: `
+              {
+                allPosts: allMdx(
+                  sort: { fields: [frontmatter___date], order: DESC }
+                  filter: { fields: { collection: { in: ["blog"] } }, frontmatter: { published: { eq: true }, } }
+                  limit: 20
+                ) {
+                  nodes {
+                    body
+                    html
+                    frontmatter {
+                      title
+                      author
+                      description
+                      date(formatString: "MMM DD YYYY")
+                      thumbnail {
+                        publicURL
+                      }
+                    }
+                    fields {
+                      collection
+                      slug
+                    }
+                  }
+                }
+              }
+            `,
+            output: "/blog/feed.xml",
+            title: "Layer5"
```

Comment:
Make it `Layer5 Blogs`

Reply:
I also merge the all feeds in one..so should I do it again after fetching new update?

Reply:
So do I need to to create a new branch for that?

Reply:
As I am seeing there is conficits

Reply:
I was just started to do that...almost on the way of completion...But thanks..😅

Reply:
Ahhh....extremely sorry for the inconvenience 😬

Reply:
😂😂..Not a problem....by the way, you like a superhero for helping people